src/hidimstat/statistical_tools/lasso_test.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+from sklearn.linear_model import LassoCV
+from sklearn.model_selection import KFold
+
+
+def preconfigure_LassoCV(estimator, X, X_tilde, y, n_alphas=20):
+    """
+    Configure the estimator for Model-X knockoffs.
```

Comment:
I think the docstring should focus more on L43-44: the regularization path is defined in a data-dependent way. 
The paragraph in Notes should be the central part of the docstring. Is there a reference?